packages/menus/src/views/__snapshots__/MenuView.spec.js.snap

```diff
@@ -948,7 +1116,7 @@ exports[`MenuView Renders renders custom top animation correctly 1`] = `
         placement="top"
       >
         <ul
-          className="c-menu c-menu--up is-open c-arrow--b c1"
+          className="c-menu c-menu--up c-arrow--b c1"
```

Comment:
I'm still seeing strange placement/behavior in Chrome.

![jul-02-2018 12-32-21](https://user-images.githubusercontent.com/143773/42182940-d695b7e8-7df4-11e8-96f3-c06e1ab1b6e6.gif)

Reply:
> is-active state as expected

This is now added to the example.

> I'm still seeing strange placement/behavior in Chrome.

Maybe I didn't publish the new example, could you run locally? I'm currently not seeing this in Chrome, Firefox, Safari, or IE11